test/e2e_node/system/types_windows.go

```diff
@@ -0,0 +1,53 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package system
+
+import "os/exec"
+
+// DockerEndpoint is the os specific endpoint for docker communication
+const DockerEndpoint = "npipe:////./pipe/docker_engine"
+
+// DefaultSysSpec is the default SysSpec for Windows
+var DefaultSysSpec = SysSpec{
+	OS: "Microsoft Windows Server 2016",
+	KernelSpec: KernelSpec{
+		Versions:  []string{`10\.[0-9]\.1439[3-9]*`, `10\.[0-9]\.144[0-9]*`, `10\.[0-9]\.15[0-9]*`, `10\.[0-9]\.2[0-9]*`}, //requires >= '10.0.14393'
```

Comment:
- `10\.[0-9]\.1439[3-9]*` will match also `10.2.1439`.
- `10\.[0-9]\.144[0-9]*` will match `10.0.144` which is lower than '10.0.14393'
- `10\.[0-9]\.15[0-9]*`, same here. will match `10.0.15`, which is lower than '10.0.14393'
- `10\.[0-9]\.2[0-9]*`, will match `10.0.200`, , which is lower than '10.0.14393'

Something that you probably want to have here is:
```go
Versions:  []string{`10\.0\.1439[3-9]`, `10\.0\.14[4-9][0-9]{2}`, `10\.0\.1[5-9][0-9]{3}`, `10\.0\.[2-9][0-9]{4}`,`10\.[1-9]+\.[0-9]+}, //requires >= '10.0.14393'
```
That would match ranges:
- 10.0.14393 -- 10.0.14399
- 10.0.14400 -- 10.0.14999
- 10.0.15000 -- 10.0.19999
- 10.0.20000 -- 10.0.99999
- 10.1.0+

Reply:
good catch!

Reply:
updated